src/pudl/io_managers.py

```diff
@@ -371,7 +371,15 @@ def load_input(self, context: InputContext) -> pd.DataFrame:
 def pudl_sqlite_io_manager(init_context) -> SQLiteIOManager:
     """Create a SQLiteManager dagster resource for the pudl database."""
     base_dir = init_context.resource_config["pudl_output_path"]
-    md = Package.from_resource_ids().to_sql()
+    md = Package.from_resource_ids(
+        excluded_etl_groups=(
```

Comment:
Is there a better solution than hard-coding the groups to exclude?

Reply:
I don't think there is right now. The ETL Groups were kind of a stopgap to organize resources that needed to be grouped together in different steps of the ETL process. We have a similar problem in the documentation build.

Maybe it would be better to have some boolean flags that were part of the `Resource` metadata that said whether they were going to go into SQLite (which could also be used to control whether we try to link to them on Datasette), whether they should be part of the documentation build, whether they're active at all or just drafts.

It seems like this information also intersects with some of the Dagster metadata though. Like it wouldn't be ideal to say "This goes into SQLite" in the `Resource` metadata, but then change to using a different `IOManager` down the line and forget to update the `Resource`.